mitmproxy/addons/modifybody.py

```diff
@@ -30,6 +31,7 @@ def configure(self, updated):
             for option in ctx.options.modify_body:
                 try:
                     spec = parse_modify_spec(option, True)
+                    logging.log(ALERT, "Cannot modify streamed bodies.")
```

Comment:
As you said, this is just duplicating - let's take it out.

Reply:
I have removed this and the corresponding import of ALERT

Reply:
Clarifying that I did a git revert